### PR TITLE
fix: deriv ez responsive issues

### PR DESCRIPTION
**Pull request by hasan-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -176,6 +176,11 @@ export const onClientEntry = () => {
     updateURLAsPerUserLanguage()
 }
 
+export const shouldUpdateScroll = () => {
```
Comment (mohsen-deriv) [inline gatsby-browser.js:179]: this is gonna cause too many issue, we shouldn't be using this.

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: src/features/components/templates/hero-banners/trade/trade-hero.module.scss [hunk 2]
```diff
@@ -22,8 +23,7 @@
         }
 
         @include breakpoints(tablet) {
-            min-block-size: 85svh;
-            inset-block-end: unset;
+            min-block-size: 82.2rem;
```
Comment (mohsen-deriv) [inline src/features/components/templates/hero-banners/trade/trade-hero.module.scss:26]: this is gonna cause issues on home page, please check it as well :pray:

**Review by mohsen-deriv — DISMISSED** [review]
please check the comments

File: gatsby-browser.js (hunk 1 above)
Comment (hasan-deriv) [reply gatsby-browser.js:179]: qa is raising issue about scrolling in all of the new approach pages. [https://app.clickup.com/t/20696747/WEBREL-382](https://app.clickup.com/t/20696747/WEBREL-382)

Comment (hasan-deriv) [reply gatsby-browser.js:179]: can we use it as temporary solution?

File: src/features/components/templates/hero-banners/trade/trade-hero.module.scss (hunk 2 above)
Comment (hasan-deriv) [reply src/features/components/templates/hero-banners/trade/trade-hero.module.scss:26]: It's different component..

**Review by mitra-deriv — DISMISSED** [review]

File: gatsby-browser.js (hunk 1 above)
Comment (mitra-deriv) [reply gatsby-browser.js]: This is removed I assume @hasan-deriv you can resolve the conversation thanks 🙏

**Review by habib-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**